packages/datasource-sql/src/introspection/introspector.ts

```diff
@@ -18,45 +23,51 @@ export default class Introspector {
   }
 
   /** Get names of all tables in the public schema of the db */
-  private static async getTableNames(sequelize: Sequelize): Promise<string[]> {
+  private static async getTableNames(sequelize: Sequelize): Promise<SequelizeTableIdentifier[]> {
     const names: ({ tableName: string } | string)[] = await sequelize
       .getQueryInterface()
       .showAllTables();
 
-    // Fixes Sequelize behavior incorrectly implemented.
-    // Types indicate that showAllTables() should return a list of string, but it
-    // returns a list of object for both mariadb & mssql
     // @see https://github.com/sequelize/sequelize/blob/main/src/dialects/mariadb/query.js#L295
-    return names.map(name => (typeof name === 'string' ? name : name?.tableName));
+    return names.map(tableIdentifier =>
```

Comment:
```suggestion
    return tableIdentifiers.map(tableIdentifier =>
```
It's weird in the current state. 👀 
Could we add a comment in there?

Reply:
I'll rename the variable and add a comment